src/prototype_creep_resources.js

```diff
@@ -280,6 +280,26 @@ Creep.prototype.pickupEnergy = function() {
     }
   }
 
+  // FIND_TOMBSTONES and get them empty first
```

Comment:
We should create a new function for it `Creep.prototype.withdrawTombstone` and call it from the `getEnergy()` function.

Reply:
i will do